src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/set_app_icon_request.cc

```diff
@@ -60,6 +60,14 @@ SetAppIconRequest::SetAppIconRequest(
     , is_icons_saving_enabled_(false) {
   const std::string path =
       application_manager_.get_settings().app_icons_folder();
+
+  if (!file_system::DirectoryExists(path)) {
+    LOG4CXX_WARN(logger_, "App icons folder doesn't exist.");
```

Comment:
@ValeriiMalkov as a suggestion, maybe it's worth to log the unexisting path here too

Reply:
@mked-luxoft what purpose for this? See next lines, please